diff --git a/arch/arm64/boot/dts/ti/k3-am625-phyboard-lyra-rdk.dts b/arch/arm64/boot/dts/ti/k3-am625-phyboard-lyra-rdk.dts
index 0085312ff39e..7ec34e82f72e 100644
--- a/arch/arm64/boot/dts/ti/k3-am625-phyboard-lyra-rdk.dts
+++ b/arch/arm64/boot/dts/ti/k3-am625-phyboard-lyra-rdk.dts
@@ -42,9 +42,9 @@ keys {
 		pinctrl-0 = <&gpio_keys_pins_default>;
 
 		key-home {
-			label = "home";
-			linux,code = <KEY_HOME>;
-			gpios = <&main_gpio1 23 GPIO_ACTIVE_HIGH>;
+			label = "F4";
+			linux,code = <KEY_F4>;
+			gpios = <&mcu_gpio0 23 GPIO_ACTIVE_LOW>;
 		};
 
 		key-menu {
@@ -147,12 +147,6 @@ AM62X_IOPAD(0x0a0, PIN_OUTPUT, 1) /* (K25) GPMC0_WPn.AUDIO_EXT_REFCLK1 */
 		>;
 	};
 
-	gpio_keys_pins_default: gpio-keys-pins-default {
-		pinctrl-single,pins = <
-			AM62X_IOPAD(0x1d4, PIN_INPUT, 7) /* (B15) UART0_RTSn.GPIO1_23 */
-		>;
-	};
-
 	hdmi_int_pins_default: hdmi-int-pins-default {
 		pinctrl-single,pins = <
 			AM62X_IOPAD(0x040, PIN_INPUT, 7) /* (N23) GPMC0_AD1.GPIO0_16 */
@@ -277,6 +271,18 @@ AM62X_IOPAD(0x088, PIN_INPUT, 7) /* (L24) GPMC0_OEn_REn.GPIO0_33 */
 	};
 };
 
+&mcu_pmx0 {
+        gpio_keys_pins_default: gpio-keys-pins-default {
+                pinctrl-single,pins = <
+                        AM62X_IOPAD(0x084, PIN_INPUT, 7) /* WKUP_CLKOUT0.MCU_GPIO0_23 */
+                >;
+        };
+};
+
+&mcu_gpio0 {
+        status = "okay";
+};
+
 &audio_refclk1 {
 	assigned-clock-rates = <19200000>;
 };
